emperor/core.py

```diff
@@ -95,10 +95,11 @@ class Emperor(object):
         A list of the OrdinationResults objects with the same sample
         identifiers as the identifiers in ``ordination``.
     ignore_missing_samples: bool, optional
-        If set to `True` samples without metadata are included by setting all
-        metadata values to: ``This sample has not metadata``. By default an
-        exception will be raised if missing samples are encountered. Note, this
-        flag only takes effect if there's at least one overlapping sample.
+        If set to `True` samples and features without metadata are included by
+        setting all metadata values to: ``This element has not metadata``. By
```

Comment:
What about "This element is missing metadata"?

Reply:
...also the actual text in the code was "This element has no metadata" which would also be okay. The "has not" was weird :)

Reply:
Got it, I'll correct the documentation then 👍